README.md

```diff
@@ -10,6 +10,7 @@
 <p align="center">
 <a href="https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/actions/workflows/release.yml"><img src="https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/actions/workflows/release.yml/badge.svg" alt="CI badge" /></a>
 <a href="https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/actions/workflows/test.yml"><img src="https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/actions/workflows/test.yml/badge.svg" alt="CI badge" /></a>
+<a href="https://fingerprintjs.github.com/fingerprint-pro-server-api-php-sdk"><img src="https://fingerprintjs.github.com/fingerprint-pro-server-api-php-sdk/coverage.svg" alt="Unit Test Coverage" /></a>
```

Comment:
You used the wrong domain.
I turned on github page deploy, so you can check the result now https://fingerprintjs.github.io/fingerprint-pro-server-api-php-sdk/

Reply:
This commit should fix this
https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/pull/103/commits/4876354026c220fe15781b96ed0d752f29846402